crates/hyperqueue/src/client/commands/job.rs

```diff
@@ -36,39 +34,33 @@ pub struct JobListOpts {
 #[derive(Parser)]
 pub struct JobInfoOpts {
     /// Single ID, ID range or `last` to display the most recently submitted job
-    pub selector_arg: SelectorArg,
+    pub selector_arg: IdSelectorArg,
 }
 
 #[derive(Parser)]
 pub struct JobTasksOpts {
     /// Job ID
     pub job_id: u32,
+
+    /// Select specific task(s) by ids and state
```

Comment:
Let's remove the documentation comments on `flatten`ed fields. The reader might think that they are shown in help, but they are not. The same in `JobCatOpts`.